inlong-sort/sort-connectors/pulsar/src/main/java/org/apache/inlong/sort/pulsar/table/DynamicPulsarDeserializationSchema.java

```diff
@@ -132,6 +140,39 @@ public void open(DeserializationSchema.InitializationContext context) throws Exc
 
     }
 
+    /**
+     * reflect get metricGroup
+     *
+     * @param context
```

Comment:
Suggested adding description for `@param` and `@return`.

Reply:
done